mobile/Navigation/NavigationBar.jsx

```diff
@@ -21,16 +16,10 @@ export default function NavigationBar() {
       <Tab.Screen name="PostSignInLanding" component={PostSignInLanding} options={{ headerShown: false }} />
       <Tab.Screen name="Feed" component={Feed} options={{ headerShown: false }} />
       <Tab.Screen name="CheckIn" component={CheckIn} options={{ headerShown: false }} />
-      <Tab.Screen name="Trends" component={TrendsPage} options={{ headerShown: true }} />
-      <Tab.Screen name="Journal" component={JournalPage} options={{ headerShown: false }} />
       <Tab.Screen name="Journal History" component={JournalHistoryPage} options={{ headerShown: false }} />
```

Comment:
i think that after some minor edits to the nav bar and to journal page, this page should also be moved to the new nav bar